source/VideoTransportSlider.js

```diff
@@ -646,10 +646,12 @@
 		* @private
 		*/
 		setValue: function(val) {
-			if(Math.abs(this.calcVariationPercent(val)) > this.smallVariation) {
+			var v = this.clampValue(this.min, this.max, val);
+			this.currentTime = v;
```

Comment:
setValue of TransportSlider calls _setValue with value.
_setValue calls updateKnobPosition with value.
_updateKnobPosition is updating slider position with value.
But, the time is updated by currentTime which is not synced with value.
So, assigning currentTime before calling this._setValue(val); in setValue to sync.